File: app/src/main/java/com/erkaslan/moviedb/MainActivity.java
package com.erkaslan.moviedb;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;

import com.erkaslan.moviedb.ui.HomeFragment;

public class MainActivity extends AppCompatActivity {

    private static final String TAG = "Main";

    public MainActivity() {

        super(R.layout.activity_main);

    }

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        /*
        if (savedInstanceState == null) {
            getSupportFragmentManager().beginTransaction()
                    .setReorderingAllowed(true)
                    .add(R.id.nav_graph, HomeFragment.class, null)
                    .commit();
        }

         */
        
        
    }

}

File: app/src/main/java/com/erkaslan/moviedb/MainActivity.kt
package com.erkaslan.moviedb

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.util.Log
import com.erkaslan.moviedb.databinding.ActivityMainBinding

class MainActivity : AppCompatActivity() {

    private lateinit var binding: ActivityMainBinding
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityMainBinding.inflate(layoutInflater)
        setContentView(binding.root)
        Log.v(MainActivity.TAG, "Main: onCreate")
    }

    companion object {
        private const val TAG = "Main"
    }

}